docs/pages/blog/bringing-consistency-to-material-ui-customization-apis.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Bringing consistency to Material UI customization APIs
+description: We're standardizing two key areas of the Material UI customization APIs to reduce complexity and cognitive overhead. Read on to learn what's changing.
+date: 2024-03-11T00:00:00.000Z
```

Comment:
Just a heads-up so we ensure to update this with the expected publishing date

Reply:
Thanks, tentatively updated to March 18th.